pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -60,6 +60,25 @@ const LOCAL_INSTALL_MEDIA_SOURCE = 'file';
 const EXISTING_DISK_IMAGE_SOURCE = 'disk_image';
 const PXE_SOURCE = 'pxe';
 
+function volumeNameToPath(volumeName, poolName, connectionName, storagePools) {
+    const pool = storagePools.find(pool => pool.name === poolName && pool.connectionName === connectionName);
+    let volume;
+    if (pool.volumes)
+        volume = pool.volumes.find(volume => volume.name === volumeName);
+
+    return volume ? volume.path : undefined;
+}
+
+function volumeUsedByAnotherVm(volumePath, vms) {
```

Comment:
I have implemented this logic already here https://github.com/cockpit-project/cockpit/blob/master/pkg/machines/components/storagePools/storagePoolVolumesTab.jsx#L77

Please extract it out into a helper function a reuse it.